### PR TITLE
Sets correct publication date for RailsConf 2012

### PR DESCRIPTION
**Pull request by nicogaldamez**
Sets correct publication date using information from https://github.com/newhavenrb/conferences/wiki/RailsConf-2012

Relates to #376

### CODE REVIEW

File: data/railsconf/railsconf-2012/videos.yml [hunk 1]
```diff
@@ -321,7 +321,7 @@
   speakers:
     - Bryan Liles
   event_name: RailsConf 2012
-  published_at: "2012-08-22"
+  published_at: "2012-03-23"
```
Comment (marcoroth) [inline data/railsconf/railsconf-2012/videos.yml:324]: I assume this should also be April?
```suggestion
  published_at: "2012-04-23"
```

**Review by marcoroth — COMMENTED** [review]
Awesome, thanks!

Would you mind adding  `https://github.com/newhavenrb/conferences/wiki/RailsConf-2012` as a comment to the YAML file too. It seems that there is other information we could reference later.

And you can also remove the TODO on L3 now 👍🏼

**Review by marcoroth — APPROVED** [review]
Thank you @nicogaldamez! 🙌🏼

**Comment by benjaminoakes** [comment]
I love seeing this wiki being used in 2024 :heart:

**Comment by ZachBeta** [comment]
New Haven Ruby ❤️

**Pull request merged**